### DISCUSSION TITLE
What should I do after I run the script?

### DISCUSSION THREAD

**Discussion by jotherthings** [chriskyfung/AutoFetcher-IG-Stories-to-GDrive#146] [Q&A]
Sorry if I ask silly question
After I run the script, what should i do? close google sheets? what happen if i close sheets? 
Though I use my burner account, what best practice if I don't want my account to get banned? only open sheets/run script when i want to download stories?

Thanks

**Comment by chriskyfung** [comment]
You can close the Google Sheets after you run the script. The Apps Script backend will keep downloading stories until the job finishes or times out.

You don't need to open the sheet file at all. You can set auto trigger or run manually from Apps Script Editor. You can find the execution log in the Apps Script panel.

To prevent being frequently blocked by Instagram bot detection, you should avoid abuse. Behave like a native user, view the IG website in the browser, from which you grab the cookies, before you run the script. Scrape only those whom you followed, just a few users and 1-2 times per day.

Please keep in mind that scraping stories may violate GDPR and/or Meta's terms. Any abuse is strictly at your own risk.
